variables.tf

```diff
@@ -0,0 +1,85 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# REQUIRED PARAMETERS
+# You must provide a value for each of these parameters.
+# ---------------------------------------------------------------------------------------------------------------------
+
+variable "name" {
+  description = "Name to be used on all the resources as identifier"
+  type        = string
+  default     = "aws-eventbridge-to-amazon-q-developer"
```

Comment:
if you have a default then it is not required variable. Required variables do not have defaults